doc/man1/flux-jobs.rst

```diff
@@ -382,6 +383,17 @@ include:
    ``all``, ``up``, ``down``, ``allocated``, or ``free``, e.g.
    ``{instance.resources.all.ncores}``
 
+The following fields may return different information depending on
+the state of the job or other context:
+
+**contextual_info**
+   Returns selected information based on the job's current state.  If the
+   job is in PRIORITY state, then the string ``priority-wait`` is returned,
+   if the job is in DEPEND state, then a list of outstanding  dependencies
+   is returned, if the job is in SCHED state then an estimated time the
+   job will run is returned (if the scheduler supports it). Otherwise,
+   the assigned nodelist is returned (if resources were assigned).
+
```

Comment:
idea: move `status` and `status_abbrev` into this area too?  fields that are "combo" fields?  There is already a `Job status` section, so not sure if it would be more confusing or not.

Reply:
I was thinking of contextual fields as displaying a different JobInfo property depending on job state or some other information. Since `status` is itself a JobInfo property, it doesn't fit into this category. (Does that even make sense?)

Reply:
`status` is sort of the "decrepit" contextual field, show `state` except when state == inactive, show `result`.  It's obviously the most minimal contextual one.

And isn't `contextual_info` also a JobInfo property?  You also added it to the class.  Or did you mean a uhh "native" properties.

Reply:
Yeah, I can see that it is actually a gray area. I was thinking of `status` as more well defined -- the job has a "status" (it is just a conglomeration of state or result), but it doesn't really have this new field which is just a convenience property for use in `flux-jobs` or other `JobInforFormat` use cases.